omnigibson/prims/joint_prim.py

```diff
@@ -469,9 +469,11 @@ def lower_limit(self):
         raw_pos_lower, raw_pos_upper = self._articulation_view.get_joint_limits(
             joint_indices=self.dof_indices
         ).flatten()
+        raw_pos_lower = th.tensor(raw_pos_lower)
+        raw_pos_upper = th.tensor(raw_pos_upper)
```

Comment:
so the articulation view is still returning numpy arrays? why not retuning torch tensors directly?